content/en/docs/setup/independent/high-availability.md

```diff
@@ -170,24 +168,40 @@ SSH is required if you want to control all nodes from a single machine.
     - `controlPlaneEndpoint` should match the address or DNS and port of the load balancer.
     - It's recommended that the versions of kubeadm, kubelet, kubectl and Kubernetes match.
 
-1.  Make sure that the node is in a clean state:
+1.  Initialize the control plane:
 
     ```sh
-    sudo kubeadm init --config=kubeadm-config.yaml
+    sudo kubeadm init --config=kubeadm-config.yaml --experimental-upload-certs
     ```
-    
+    - The `--experimental-upload-certs` flags is used to upload the control plane
+  certificates to the cluster. 
+
     You should see something like:
     
     ```sh
     ...
-    You can now join any number of machines by running the following on each node
-    as root:
+    You can now join any number of control-plane node running the following command on each as a root:
+      kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866 --experimental-control-plane --certificate-key f8902e114ef118304e561c3ecd4d0b543adc226b7a07f675f56564185ffe0c07
     
-    kubeadm join 192.168.0.200:6443 --token j04n3m.octy8zely83cy2ts --discovery-token-ca-cert-hash    sha256:84938d2a22203a8e56a787ec0c6ddad7bc7dbd52ebabc62fd5f4dbea72b14d1f
+    Please note that the certificate-key gives access to cluster sensitive data, keep it secret!
+    As a safeguard, uploaded-certs will be deleted in two hours; If necessary, you can use kubeadm init phase upload-certs to reload certs afterward.
+    
+    Then you can join any number of worker nodes by running the following on each as root:
+      kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866
     ```
 
-1.  Copy this output to a text file. You will need it later to join other control plane nodes to the
-    cluster.
+    - Copy this output to a text file. You will need it later to join control plane and worker nodes to the cluster.
+    - When `--experimental-upload-certs` is used with `kubeadm init`, the certificates of the primary control plane
+    are uploaded to the cluster in a Secret and encrypted.
+    -  When joining new control plane nodes using `kubeadm join ... --experimental-control-plane`,
```

Comment:
Please move this point after the `kubeadm join --experimental-control-plane` command (L256 before your changes)